zeus/optimizer/__init__.py

```diff
@@ -15,3 +15,4 @@
 """A collection of optimizers for various knobs."""
 
 from zeus.optimizer.power_limit import GlobalPowerLimitOptimizer
+from zeus.optimizer.power_limit import HFGPLO
```

Comment:
Please run `bash scripts/lint.sh` to automatically format code and fix any lint errors. That's why CI is failing.